azure-pipelines.yml

```diff
@@ -15,6 +15,10 @@ variables:
   value: 5.0.2
 - name: DOTNET_SKIP_FIRST_TIME_EXPERIENCE
   value: true
+- name: DOTNET_VERSION
+  value: 3.0.100
+- name: winVmImage
+  value: Hosted Windows 2019 with VS2019
```

Comment:
This will keeps us from updating let's say for XamarinForms (private agents) i prefer we had this as settable from the pipeline

Reply:
Yea once we are happy with the PR I was going to delete this before merging it.

Though I still think this should be part of the yml file not the pipeline. For example if branch 4.1.0 is set to VS 2017 than when you run those builds it'll just build and work. Then when you run 4.2.0 it'll just build and work. If it's settable at the pipeline level than you are required to have knowledge external to the repository in order to make something compile.

But we can discuss that more on Thursday :-)

Reply:
we can delete i m happy with it :P Let's merge

Reply:
Sounds good!! I want to run some quick tests on UWP to see if incrementing the SDK build will break the icon on MDP even more or if the issue is the same.

Reply:
@rmarinho I added these back in for the time being because FormsQueue and FormsStack uses C# features that need 2019. I don't really want to modify them so they compile with C# 7.0 I'd rather just leave them untouched.

So I figure once this is merged in and the build server pipeline variables are updated then we can remove the fields?

```
Xamarin.Forms.Core\corefx\FormsQueue.cs(263,19): Error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
Xamarin.Forms.Core\corefx\FormsQueue.cs(278,14): Error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
Xamarin.Forms.Core\corefx\FormsQueue.cs(285,19): Error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
Xamarin.Forms.Core\corefx\FormsStack.cs(344,23): Error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
Xamarin.Forms.Core\corefx\FormsStack.cs(374,24): Error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
Xamarin.Forms.Core\corefx\FormsStack.cs(404,23): Error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
Xamarin.Forms.Core\corefx\FormsQueue.cs(263,19): Error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
Xamarin.Forms.Core\corefx\FormsQueue.cs(278,14): Error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
Xamarin.Forms.Core\corefx\FormsQueue.cs(285,19): Error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
Xamarin.Forms.Core\corefx\FormsStack.cs(344,23): Error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
```